packages/terra-form-textarea/src/TextareaField.jsx

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import IconError from 'terra-icon/lib/icon/IconError';
 import Field from 'terra-form-field';
-
```

Comment:
We'll want to keep this space between the imports from packages and imports from relative files, theres a lint rule for it.

Reply:
I went ahead and updated this in this commit: https://github.com/cerner/terra-core/pull/2560/commits/28d9ae7f70f5b8fe15d48f3a2c00ed036d8e7612